src/app/frontend/chrome/component.js

```diff
@@ -33,16 +33,14 @@ export class ChromeController {
    */
   constructor($state, $timeout, $transitions, $resource, $sce) {
     /**
-     * By default this is true to show loading spinner for the first page.
      * @export {boolean}
      */
-    this.showLoadingSpinner = true;
+    this.showLoadingSpinner = false;
```

Comment:
Since a state change may not happen after chrome component init, we better not to set loading to true by default